PLATEXP-1812: Add RAM to a server or hypervisor (#1430)
rackerlabs/support-how-to: content/how-to/dedicated-hosting/add-ram-to-server-or-hypervisor/index.md

type: article
created_date: '2019-12-12'
created_by: Chad Sterling

> last_modified_date: '2023-02-23'

Author: Date update

1. Log in to the [MyRackspace Portal](https://login.rackspace.com/login) with your username and
   password.
   

>    <img width="409" alt="logincredentials" src="/support/how-to/add-ram-to-server-or-hypervisor/addramserverhypervisor1.png">

Author: Alt tag information is updated. Sync the branch.

> 2. In the top navigation bar, choose **Support** > **Support Center**.

Reviewer: ```suggestion
2. In the top navigation bar, click **Support** > **Support Center**.
```

Author: Done

   <img width="941" alt="selectsupportcenter" src="/support/how-to/add-ram-to-server-or-hypervisor/addramserverhypervisor2.png">

> 3. Under **Common Request** menu, click **See All Templates**. The **See All Templates** displays list of all templates.

Reviewer: ```suggestion
3. In **Common Request** section, click **See All Templates** and it will display list of all templates.
```

Author: Done

   <img width="958" alt="templatelist" src="/support/how-to/add-ram-to-server-or-hypervisor/addramserverhypervisor3.png">

> 4. From the **Common Request** menu, select **Add RAM to Server or Hypervisor** as shown in following image.

Reviewer: ```suggestion
4. From the **Common Request** window, select **Add RAM to Server or Hypervisor**.
```

Author: Done.

   <img width="894" alt="selectrequiredtemplate" src="/support/how-to/add-ram-to-server-or-hypervisor/addramserverhypervisor4.png"> 

> 5. Select **Device(s)** from the drop down list. Enter the amount of RAM you want to add as shown in following image.

Reviewer: ```suggestion
5. Select **Device(s)** from the drop down list. Enter the amount of RAM you want to add.
```

Author: Done
